api/disruption_kind.go

```diff
@@ -45,5 +45,12 @@ func AppendCommonArgs(args []string, level chaostypes.DisruptionLevel, container
 		args = append(args, "--on-init")
 	}
 
+	// append allowed hosts for network disruptions
+	if kind == chaostypes.DisruptionKindNetworkDisruption {
```

Comment:
There's a bit of a code smell putting a kind specific arg here under `AppendCommonArgs`. I see why you had to do it, but I would almost rather add `--allowed-hosts` as a flag to the injector root command, and then just pass this into all injection kinds, keeping this function true to its needs. I'll try to think of a better solution while I read the code

EDIT: After thinking I really couldn't find a better idea. my suggestion ^ isn't good, so perhaps we just rename this function and change its comment, leaving your implemented behavior?

Reply:
I agree. I changed the function name and comment, let me know if it sounds ok now.